calico_node/Dockerfile

```diff
@@ -31,6 +31,18 @@ RUN apk add --update curl && \
 ADD build.sh /build.sh
 RUN /build.sh # 23MAR2016
 
+# Metadata
```

Comment:
Can you elaborate on the "Metadata" here, with a reference to http://label-schema.org/ ?